src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -9754,7 +9754,7 @@ public static void M()
 }";
             var expectedDiagnostics = new[]
             {
-                // (10,35): error CS1525: Invalid expression term '{'
+                    // (10,35): error CS1525: Invalid expression term '{'
                 //         var b = a with { Nested = { A = 20 } };
```

Comment:
The alignment change doesn't look intentional

Reply:
Fixed.